src/toRedactor.tsx

```diff
@@ -399,7 +450,16 @@ export const toRedactor = (jsonValue: any,options?:IJsonToHtmlOptions) : string
           if (!attrsJson['sys-style-type']) {
             attrsJson['sys-style-type'] = String(allattrs['asset-type']).indexOf('image') > -1 ? 'display' : 'download'
           }
-
+          if (attrsJson?.["display-type"] === "display") {
+            const styleObj = jsonValue?.["attrs"]?.["style"] ?? {};
+            if (!styleObj["width"]) {
+              styleObj["width"] = "auto";
+            }
+            delete styleObj["float"];
+            attrsJson["style"]
```

Comment:
Can you confirm that attrsJson["style"] is always a string? because if its an object or something else then we can get weird behaviour when appending to it.

Reply:
I have added a check to make sure that the type is string